src/stores/editNetworkStore.ts

```diff
@@ -0,0 +1,23 @@
+import { action, computed, observable } from 'mobx';
+
+export class EditNetworkStore {
```

Comment:
Is this necessary to have separate from NetworkStore?

Reply:
Hmm, i don't even think we need this store anymore. But in the future I was planning to use it. Removing it now